src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -6121,10 +6149,10 @@ def create_vnet_gateway(cmd, resource_group_name, virtual_network_gateway_name,
 
     client = network_client_factory(cmd.cli_ctx).virtual_network_gateways
     subnet = virtual_network + '/subnets/GatewaySubnet'
-    active_active = len(public_ip_address) == 2
+    active = len(public_ip_address) == 2
     vnet_gateway = VirtualNetworkGateway(
         gateway_type=gateway_type, vpn_type=vpn_type, vpn_gateway_generation=vpn_gateway_generation, location=location,
-        tags=tags, sku=VirtualNetworkGatewaySku(name=sku, tier=sku), active_active=active_active, ip_configurations=[],
+        tags=tags, sku=VirtualNetworkGatewaySku(name=sku, tier=sku), active=active, ip_configurations=[],
```

Comment:
Why property name changed? Is it normal for Track 2 SDK?

Reply:
No, most are not changed. I think it's due to track2 codegen logic change.